src/pages/project/new_project/EditProject.tsx

```diff
@@ -0,0 +1,16 @@
+import { useParams } from 'react-router-dom';
+import { StyledPageContent } from '../../../components/styled/Wrappers';
+import { IdentifierParams } from '../../../utils/urlPaths';
+import ProjectForm from './ProjectForm';
+
+const EditProject = () => {
+  const { identifier } = useParams<IdentifierParams>();
```

Comment:
Det er vel hele URIen vi henter her, ikke bare UUIDen(?)
```suggestion
  const { id } = useParams<IdParams>();
```

Reply:
Finner ikke noe IdParams

Reply:
Tenkte vi kunne lage en ny for det, for å ha et konsekvent skille med at vi bruker `id` når det er en URI, og `identifier` når det er bare UUID